tests/ui/lang-items/start_lang_item_args.rs

```diff
@@ -1,14 +1,13 @@
+//@ add-core-stubs
 //@ check-fail
 //@ revisions: missing_all_args missing_sigpipe_arg missing_ret start_ret too_many_args
 //@ revisions: main_ty main_args main_ret argc argv_inner_ptr argv sigpipe
 
 #![feature(lang_items, no_core)]
 #![no_core]
 
-#[lang = "copy"]
-pub trait Copy {}
-#[lang = "sized"]
-pub trait Sized {}
+extern crate minicore;
+use minicore::*;
```

Comment:
Problem: similarly, I think this test is exercising presence/absence of lang items, so I feel like it should not be using `minicore`.

Reply:
Will change back to non-minicore.